app/components/articleSearch/components/filterContainer/index.tsx

```diff
@@ -9,70 +9,62 @@ export interface IFilterContainerProps {
   journalIFFilterValue: number;
 }
 
-const FilterContainer = (props: IFilterContainerProps) => {
```

Comment:
I think using props logic is much better. because it's easier to make other functions.

Reply:
I agree